diff --git a/pango/pango-utils.c b/pango/pango-utils.c
index 44ec353..c31c836 100644
--- a/pango/pango-utils.c
+++ b/pango/pango-utils.c
@@ -701,6 +701,8 @@ pango_get_sysconf_subdirectory (void)
       gchar *root = g_win32_get_package_installation_directory_of_module (pango_dll);
       result = g_build_filename (root, "etc\\pango", NULL);
       g_free (root);
+#elif defined __PLAYBOOK__
+      result = "app/native/etc/pango";
 #else
       const char *sysconfdir = g_getenv ("PANGO_SYSCONFDIR");
       if (sysconfdir != NULL)
@@ -740,6 +742,8 @@ pango_get_lib_subdirectory (void)
       else
 	result = g_build_filename (root, "lib\\pango", NULL);
       g_free (root);
+#elif defined __PLAYBOOK__
+      result = "app/native/lib/pango";
 #else
       const char *libdir = g_getenv ("PANGO_LIBDIR");
       if (libdir != NULL)
